app/types/entities/task.ts

```diff
@@ -17,6 +19,16 @@ export default interface Task {
    */
   orderScore: number;
 
+  /**
+   * Id of task status
+   */
+  statusId?: string;
+
+  /**
+   * Task status object
+   */
+  status?: Status|null;
```

Comment:
looks strange that we have both `statusId` and `status` fields.

Reply:
nested status object helps to display statuses for tasks in All projects view. It allows to reduce number of requests performed

Reply:
Maybe it is related to the Project properties? You can request them all at once when Project is open. And for the "All Projects" you can request them once two

Reply:
I mean something like `project.availableTaskStatuses`

Reply:
did the change